### DISCUSSION TITLE
Error code when setting featured image

### DISCUSSION THREAD

**Discussion by MakaylaBaughman** [CollectionBuilder/collectionbuilder.github.io#115] [Q&A]
Hi! I am very new to Collection Builder and website coding in general, so please keep that in mind when formulating responses (help for dummies). 
I am currently building my repository and am attempting to set a new featured image. I uploaded the image to my objects_ folder and followed the tutorials for CSV for creating my metadata and inputting the objectid in theme.yml. However, when I attempt to run the “bundle exec jekyll serve” command to generate my site, I get the following error:

Error cb_helpers: Item for featured image with objectid 'leighhuntimage' not found in configured metadata 'demo-repository'. Please check 'featured-image' in '_data/theme.yml'
          Conflict: The following destination is shared by multiple files.
                    The written file may end up with unexpected contents.

I have checked and double-checked everything according to Collection Builder instructions, and still receive the same error each time. 
If anyone has encountered this error or has a solution, please lmk!

**Comment by dcnb** [comment]
Hi @MakaylaBaughman, 

It looks like your _config.yml file needs to be revised to connect the metadata sheet you want. So edit the value for `metadata` so that it reads "projectmetadata" rather than "demo-repository" as it does now. 

You'll also want to move the objects folder back into its normal place at the root of the directory. It's currently in your /pages/ folder. Some files have also been copied into your utilities folder that don't need to be there. My suggestion would be to start a new clean CSV template, then copy the edits you've made to that new /data/theme.yml file and _config.yml file (but make sure you put projectmetadata rather than demo-repository down for the metadata value!). And then move your metadata over. 

You might just go through the CSV walkthrough again as well, starting at [step 3](https://collectionbuilder.github.io/cb-docs/docs/walkthroughs/csv-walkthrough/#3-create-a-new-repository). Then just use your metadata instead of the demo metadata!  

Good luck, 
@dcnb

**Reply by MakaylaBaughman** [reply]
Thank you so much, that makes perfect sense!
